tensorflow/tools/dist_test/scripts/dist_mnist_test.sh

```diff
@@ -147,7 +147,7 @@ done
 # Get N_WORKERS by WORKER_HOSTS
 N_WORKERS=$(echo ${WORKER_HOSTS} | awk -F "," '{printf NF}')
 # Replace the delimiter with " "
-WORKER_ARRAY=$(echo ${WORKER_HOSTS} | awk -F "," '{for(i=1;i<=NF;i++){printf $i" "}}')
+WORKER_ARRAY=($(echo ${WORKER_HOSTS} | awk -F "," '{for(i=1;i<=NF;i++){printf $i" "}}'))
```

Comment:
The right way for assign bash array: `array=($string)`. It's obvious that the old `WORKER_ARRAY` gets a string instead of an array. The same as `PS_ARRAY`.